homeassistant/components/sensor/modbus.py

```diff
@@ -40,6 +41,7 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_REGISTERS): [{
+        vol.Required(CONF_HUB, DEFAULT_HUB): cv.string,
```

Comment:
Please use keyword argument `default` for default value.

Reply:
Since we set a default we can make this optional.

Reply:
Made hub name optional.